src/catalog/postgres/builder.cpp

```diff
@@ -651,92 +653,92 @@ Schema Builder::GetProcTableSchema() {
-  columns.back().SetOid(PROROWS_COL_OID);
+  columns.back().SetOid(PgProc::PROROWS_COL_OID);
 
   columns.emplace_back("provariadic", type::TypeId::INTEGER, false,
                        parser::ConstantValueExpression(type::TypeId::INTEGER));
-  columns.back().SetOid(PROVARIADIC_COL_OID);
+  columns.back().SetOid(PgProc::PROVARIADIC_COL_OID);
 
   columns.emplace_back("proisagg", type::TypeId::BOOLEAN, false,
                        parser::ConstantValueExpression(type::TypeId::BOOLEAN));
-  columns.back().SetOid(PROISAGG_COL_OID);
+  columns.back().SetOid(PgProc::PROISAGG_COL_OID);
 
   columns.emplace_back("proiswindow", type::TypeId::BOOLEAN, false,
                        parser::ConstantValueExpression(type::TypeId::BOOLEAN));
-  columns.back().SetOid(PROISWINDOW_COL_OID);
+  columns.back().SetOid(PgProc::PROISWINDOW_COL_OID);
 
   columns.emplace_back("proisstrict", type::TypeId::BOOLEAN, false,
                        parser::ConstantValueExpression(type::TypeId::BOOLEAN));
-  columns.back().SetOid(PROISSTRICT_COL_OID);
+  columns.back().SetOid(PgProc::PROISSTRICT_COL_OID);
 
   columns.emplace_back("proretset", type::TypeId::BOOLEAN, false,
                        parser::ConstantValueExpression(type::TypeId::BOOLEAN));
-  columns.back().SetOid(PRORETSET_COL_OID);
+  columns.back().SetOid(PgProc::PRORETSET_COL_OID);
 
   columns.emplace_back("provolatile", type::TypeId::BOOLEAN, false,
                        parser::ConstantValueExpression(type::TypeId::BOOLEAN));
-  columns.back().SetOid(PROVOLATILE_COL_OID);
+  columns.back().SetOid(PgProc::PROVOLATILE_COL_OID);
 
   columns.emplace_back("pronargs", type::TypeId::SMALLINT, false,
                        parser::ConstantValueExpression(type::TypeId::TINYINT));
-  columns.back().SetOid(PRONARGS_COL_OID);
+  columns.back().SetOid(PgProc::PRONARGS_COL_OID);
 
   columns.emplace_back("pronargdefaults", type::TypeId::SMALLINT, false,
                        parser::ConstantValueExpression(type::TypeId::TINYINT));
-  columns.back().SetOid(PRONARGDEFAULTS_COL_OID);
+  columns.back().SetOid(PgProc::PRONARGDEFAULTS_COL_OID);
 
   columns.emplace_back("prorettype", type::TypeId::INTEGER, false,
                        parser::ConstantValueExpression(type::TypeId::INTEGER));
-  columns.back().SetOid(PRORETTYPE_COL_OID);
+  columns.back().SetOid(PgProc::PRORETTYPE_COL_OID);
 
   columns.emplace_back("proargtypes", type::TypeId::VARBINARY, 4096, false,
                        parser::ConstantValueExpression(type::TypeId::VARBINARY));
-  columns.back().SetOid(PROARGTYPES_COL_OID);
+  columns.back().SetOid(PgProc::PROARGTYPES_COL_OID);
 
   columns.emplace_back("proallargtypes", type::TypeId::VARBINARY, 4096, false,
                        parser::ConstantValueExpression(type::TypeId::VARBINARY));
-  columns.back().SetOid(PROALLARGTYPES_COL_OID);
+  columns.back().SetOid(PgProc::PROALLARGTYPES_COL_OID);
```

Comment:
Does not follow semantics of Postgres' column.  From the catalog [documentation](https://www.postgresql.org/docs/12/catalog-pg-proc.html)

>  [...] however, if all the arguments are IN arguments, this field will be **null**.[...]

Specifically, this column should be `NULL` for the vast majority of standard functions in the table which ties back to a logic bug in the implementation.

Reply:
I'm fine with this being resolved by creating a "bug" issue for the behavior since it probably involves significant tweaks to logic elsewhere.

Reply:
Added a comment in PgProcImpl::CreateProcedure referencing #1359.